Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -9,14 +8,41 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class CaesarCipher
     {
+        const int CypherLength = 95; // number of characters before it loops to the start
+        const int Unusable = 32; //the first 32 chars of ascii are not printable
```

Comment:
Without your comment this is highly confusing. IndexOfLastUnusable or something like that would make more sense.

Reply:
Changed Unusable to IndexOfLastUnusable